src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics/Generated/Models/SubResource.cs

```diff
@@ -18,13 +18,12 @@ namespace Microsoft.Azure.Management.DataLake.Analytics.Models
     using Analytics;
     using Newtonsoft.Json;
     using Rest;
-    using Rest.Azure;
     using System.Linq;
 
     /// <summary>
     /// The Sub Resource model definition.
     /// </summary>
-    public partial class SubResource : IResource
```

Comment:
This shouldn't be a breaking change since IResource is empty. We got a bug on this from the ruby dev, indicating that our definition of a SubResource is not the same as the true Azure one, so it was causing breaks in other languages. For C# IResource is an empty class. Additionally, SubResources cannot be created with just an IResource since they have required parameters for us, so there should be no breaks.